mmdeploy/codebase/mmagic/deploy/super_resolution_model.py

```diff
@@ -65,8 +65,8 @@ def _init_wrapper(self, backend: Backend, backend_files: Sequence[str],
             **kwargs)
 
     def convert_to_datasample(
-            self, predictions: EditDataSample, data_samples: EditDataSample,
-            inputs: Optional[torch.Tensor]) -> List[EditDataSample]:
+            self, predictions: DataSample, data_samples: DataSample,
```

Comment:
`predictions` is already a DataSample. So the name `convert_to_datasample` is not appropriate. Maybe we can pass `pred_img` as an argument or we can rename the function.

Reply:
MMagic iterate the `forward` output like:
```python
for data_sample in data_samples:
    pred_img = data_sample.pred_img
    ...
```
So how about we rename this function from `convert_to_datasample` to `wrap_in_datasample`?

Reply:
@RunningLeon any suggestion? Anyway, the docstring of the function should be updated.